front/src/modules/ui/editable-field/components/EditableField.tsx

```diff
@@ -20,9 +20,9 @@ const StyledIconContainer = styled.div`
   svg {
     align-items: center;
     display: flex;
-    height: 16px;
+    height: 12px;
     justify-content: center;
-    width: 16px;
+    width: 12px;
```

Comment:
Hi, thank you for your contribution!

I think the "sm" size mentioned in the issue refers to the theme variable. So you could use for both height and width values:

```
${({ theme }) => theme.icon.size.sm}px
```

Reply:
Sorry @Rishy-2021, I just talked with @Bonapara and it seems the issue contains an error: it was meant to be the icon stroke, not size. The icon stroke should be `theme.icon.stroke.sm` instead of `md`.

Reply:
Following @jenalla's comment: there is also a `theme.icon.stroke.sm` theme variable, you could use it to set the `stroke-width` css property instead!